docker-compose.yml

```diff
@@ -22,6 +22,7 @@ services:
       - --accesslog=true
     ports:
       - 80:80
```

Comment:
I think we can remove 80 and use only 8080, WDYT?

Reply:
Might be a little bit inconvenient 🤔

Reply:
We could consider that, but it would not be related to Gitpod support, so we can have that discussion later.. I agree with the idea of not using system ports, it was the first thing that I noticed with Appwrite... I had some nginx server running on my PC, and I had to always stop it because Appwrite (one app) wanted to use it.